File: CS-Examples/04_Sections/RemoveSectionContent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;
using Spire.Doc.Fields;

namespace RemoveSectionContent
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new instance of the Document class.
            Document doc = new Document();

            // Load a Word document from a specified file path using the LoadFromFile method.
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\Template_N3.docx");

            // Iterate through each section in the document.
            foreach (Section section in doc.Sections)
            {
                // Clear the child objects in the header of the current section.
                section.HeadersFooters.Header.ChildObjects.Clear();

                // Clear the child objects in the body of the current section.
                section.Body.ChildObjects.Clear();

                // Clear the child objects in the footer of the current section.
                section.HeadersFooters.Footer.ChildObjects.Clear();
            }

            // Specify the output file name for the section content removal document.
            string output = "RemoveSectionContent_out.docx";

            // Save the document to a file with the specified output file name and Docx2013 format.
            doc.SaveToFile(output, FileFormat.Docx2013);

            // Release system resources used by the document.
            doc.Dispose();

            //Launch the file
            FileViewer(output);
        }
        private void FileViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/04_Sections/RemoveSectionContent.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Fields

Namespace RemoveSectionContent
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create a new Document object called doc
			Dim doc As New Document()

			'Load a Word document from the specified file path
			doc.LoadFromFile("..\..\..\..\..\..\Data\Template_N3.docx")

			'Iterate through each Section in the document
			For Each section As Section In doc.Sections

				'Clear the child objects in the header of the current section
				section.HeadersFooters.Header.ChildObjects.Clear()

				'Clear the child objects in the body of the current section
				section.Body.ChildObjects.Clear()

				'Clear the child objects in the footer of the current section
				section.HeadersFooters.Footer.ChildObjects.Clear()
			Next section

			'Specify the output file name
			Dim output As String = "RemoveSectionContent_out.docx"

			'Save the modified document as a Word document with the specified file format
			doc.SaveToFile(output, FileFormat.Docx2013)

			'Dispose of the doc object to release resources
			doc.Dispose()

			'Launch the file
			FileViewer(output)
		End Sub
		Private Sub FileViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
